scan/src/components/UserAccount.re

```diff
@@ -0,0 +1,152 @@
+module Styles = {
+  open Css;
+
+  let connectBtn =
+    style([
+      backgroundColor(Colors.green1),
+      padding2(~h=`px(8), ~v=`px(2)),
+      display(`flex),
+      justifyContent(`center),
+      alignItems(`center),
+      borderRadius(`px(10)),
+      cursor(`pointer),
+      boxShadow(Shadow.box(~x=`zero, ~y=`px(4), ~blur=`px(4), rgba(17, 85, 78, 0.1))),
+    ]);
+
+  let disconnectBtn =
+    style([
+      backgroundColor(Colors.yellowAccent1),
+      padding2(~h=`px(8), ~v=`px(2)),
+      display(`flex),
+      justifyContent(`center),
+      alignItems(`center),
+      borderRadius(`px(10)),
+      cursor(`pointer),
+      boxShadow(Shadow.box(~x=`zero, ~y=`px(4), ~blur=`px(4), rgba(99, 81, 3, 0.1))),
+    ]);
+
+  let faucetBtn =
+    style([
+      backgroundColor(Colors.blue1),
+      padding2(~h=`px(8), ~v=`px(2)),
+      display(`flex),
+      justifyContent(`center),
+      alignItems(`center),
+      borderRadius(`px(10)),
+      cursor(`pointer),
+      boxShadow(Shadow.box(~x=`zero, ~y=`px(4), ~blur=`px(4), rgba(11, 29, 142, 0.1))),
+    ]);
+
+  let logo = style([width(`px(10))]);
+
+  let balanceContainer = style([display(`flex), alignItems(`center)]);
+};
+
+module ConnectBtn = {
+  [@react.component]
+  let make = (~connect) => {
+    <div className=Styles.connectBtn onClick={_ => connect()}>
+      <Text
+        value="connect"
+        size=Text.Xs
+        weight=Text.Medium
+        color=Colors.green7
+        nowrap=true
+        height={Text.Px(10)}
+        spacing={Text.Em(0.03)}
+        block=true
+      />
+      <HSpacing size=Spacing.sm />
+      <img src=Images.connectIcon className=Styles.logo />
+    </div>;
+  };
+};
+
+module DisconnectBtn = {
+  [@react.component]
+  let make = (~disconnect) => {
+    <div className=Styles.disconnectBtn onClick={_ => disconnect()}>
+      <Text
+        value="disconnect"
+        size=Text.Xs
+        weight=Text.Medium
+        color=Colors.yellowAccent7
+        nowrap=true
+        height={Text.Px(10)}
+        spacing={Text.Em(0.03)}
+        block=true
+      />
+      <HSpacing size=Spacing.sm />
+      <img src=Images.disconnectIcon className=Styles.logo />
+    </div>;
+  };
+};
+
+module FaucetBtn = {
+  [@react.component]
+  let make = () => {
+    <div className=Styles.faucetBtn>
```

Comment:
Let's alert or console.log after in onClick for now.

Reply:
Sure